hedera-mirror-rest-java/src/test/java/com/hedera/mirror/restjava/repository/NftAllowanceRepositoryTest.java

```diff
@@ -21,16 +21,77 @@
 import com.hedera.mirror.restjava.RestJavaIntegrationTest;
 import lombok.RequiredArgsConstructor;
 import org.junit.jupiter.api.Test;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.domain.Sort.Direction;
 
 @RequiredArgsConstructor
 class NftAllowanceRepositoryTest extends RestJavaIntegrationTest {
 
     private final NftAllowanceRepository nftAllowanceRepository;
 
     @Test
-    void save() {
+    void findBySpenderAndFilterByOwnerGtAndTokenGt() {
         var nftAllowance = domainBuilder.nftAllowance().get();
         nftAllowanceRepository.save(nftAllowance);
-        assertThat(nftAllowanceRepository.findById(nftAllowance.getId())).get().isEqualTo(nftAllowance);
+        nftAllowanceRepository.save(domainBuilder.nftAllowance().get());
+        nftAllowanceRepository.save(domainBuilder.nftAllowance().get());
```

Comment:
Same for all other calls to save.
```suggestion
        domainBuilder.nftAllowance().persist();
        domainBuilder.nftAllowance().persist();
```